src/core/include/units/bits/external/fixed_string.h

```diff
@@ -23,7 +23,6 @@
 #pragma once
 
 #include <units/bits/external/hacks.h>  // IWYU pragma: keep
-// TODO use <algorithm> when moved to C++20 modules (parsing takes too long for each translation unit)
```

Comment:
`#if UNITS_MODULES`?

Reply:
I guess it works by coincidence. Before some refactoring, an `import <algorithm>;` used to be visible.

Reply:
I meant that the `algorithm` header should be only included/imported then `UNITS_MODULES` is `true`, right?

Reply:
That's right.

Reply:
I left this alone for now.